gradle/protobuf.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'com.google.protobuf'
+
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc:3.21.2'
+    }
+}
+
+dependencies {
+    api 'com.google.protobuf:protobuf-java'
```

Comment:
Does this need to be api? We had this as implementation before.

Reply:
Yes, `com.google.protobuf` are exposed in multiple places in the generated classes. For example, `StreamMetadataOrBuilder.getHash`.